spec/expression.dd

```diff
@@ -1906,6 +1906,16 @@ $(GNAME ValueExpression):
         a $(I ValueSeq), then the elements of the $(I ValueSeq)
         are inserted as arguments in place of the sequence.
     )
+    
+    $(P If there are duplicate keys in the $(I KeyValuePairs), then
+        the value returned for that key is going to be that of the
+        last $(I KeyValuePair) declared.
+    )
+    
+        ---
+        auto aa = [21u: "he", 38: "ho", 2: "hi", 2:"bye"];
```

Comment:
```suggestion
        auto aa = [21: "he", 38: "ho", 2: "hi", 2:"bye"];
```

We need the reader to focus on the important aspect. Having 21 as unsigned in this example is just noise.